pkgs/development/php-packages/openswoole

```diff
@@ -0,0 +1,23 @@
+{ lib, stdenv, buildPecl, php, valgrind, pcre2 }:
+
+buildPecl {
+  pname = "openswoole";
+
+  version = "4.11.0";
+  sha256 = "sha256-8YQN7exDbVGHKUD9e0bPrJfbHubv3Jb9rZqCPfja52g=";
+
+  buildInputs = [ pcre2 ] ++ lib.optionals (!stdenv.isDarwin) [ valgrind ];
+  internalDeps = lib.optionals (lib.versionOlder php.version "7.4") [ php.extensions.hash ];
+
+  doCheck = true;
+  checkTarget = "tests";
+
+  meta = with lib; {
+    description = "Coroutine-based concurrency library and high performance programmatic server for PHP";
+    longDescription = "Open Swoole allows you to build high-performance, async multi-tasking webservices and applications using an easy to use Coroutine API.\nOpen Swoole is a complete async solution that has built-in support for async programming via coroutines.\nIt offers a range of multi-threaded I/O modules (HTTP Server, WebSockets, TaskWorkers, Process Pools) out of the box and support for popular PHP clients like PDO for MySQL, and CURL.\nYou can use the sync or async, Coroutine API to write whole applications or create thousands of light weight Coroutines within one Linux process.";
+    homepage = "https://www.openswoole.com/";
+    license = licenses.asl20;
+    maintainers = teams.php.members;
+    changelog = "https://pecl.php.net/package/openswoole/${version}";
```

Comment:
Could you please sort alphabetically the keys ?

Reply:
Sorry about that, should be done now.